### ISSUE TITLE
dwyl week @ Founders & Coders

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/hq#295]
Saw this in an email! What is it @iteles ?

**Comment by iteles** [comment]
This one is @nelsonic !

**Comment by iteles** [comment]
Closing in favour of https://github.com/foundersandcoders/london-curriculum/issues/15

**Comment by nelsonic** [comment]
@markwilliamfirth the _idea_ behind having a dwyl week @ F&C was to get the people to go through the dwyl "***on-boarding exercise***" and give us a _ton_ of feedback in the process.
What is the dwyl "***on-boarding exercise***" you ask?
Simple: build something _useful_ to someone else in a week from problem statement to prototype with UX testing and feedback and summarise it as a case study.
I intend to _fully_ elaborate on this before the week starts with the current F&C cohort.
Waiting on responses to my questions on: https://github.com/foundersandcoders/london-curriculum/issues/15#issuecomment-301226643 as want to _tailor_ it well. 
Also buys me time to fill in some gaps in our knowledgebase _before_ I dedicate time to it. 👍
